.github/workflows/tests.yaml

```diff
@@ -8,11 +8,10 @@ jobs:
             matrix:
                 node-version:
                 - 18.x
-                - 22.x
                 - latest
         steps:
         - uses: actions/checkout@v4
-        - uses: actions/setup-node@v4
+        - uses: actions/setup-node@v3
```

Comment:
This looks wrong; is the entire commit perhaps obsolete with b7cfc4fa24c9b51be8c4f843408c780a2d9d96b8?

Reply:
You are absolutely right. Shows again the value of reviews, even for small changes :bow: